toolchain/mfc/run/run.py

```diff
@@ -102,7 +102,7 @@ def __generate_job_script(targets, case: input.MFCInputFile):
         case=case,
         MFC_ROOTDIR=MFC_ROOTDIR,
         qsystem=queues.get_system(),
-        profiler=__profiler_prepend(),
+        profiler=' '.join(__profiler_prepend()),
```

Comment:
This should probably become:
```python
profiler=' '.join([ shlex.quote(_) for _ in __profiler_prepend()])
```
so that arguments that contain whitespace are not split, and those with special characters are not misinterpreted.